legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/jdbc/SnowflakeJdbcHelper.java

```diff
@@ -0,0 +1,160 @@
+package org.finos.legend.engine.persistence.components.relational.snowflake.jdbc;
+
+import org.finos.legend.engine.persistence.components.executor.TabularData;
+import org.finos.legend.engine.persistence.components.relational.jdbc.JdbcHelper;
+import org.finos.legend.engine.persistence.components.relational.jdbc.JdbcTransactionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class SnowflakeJdbcHelper extends JdbcHelper
+{
+	private static final Logger LOGGER = LoggerFactory.getLogger(SnowflakeJdbcHelper.class);
+
+	private SnowflakeJdbcHelper(Connection connection)
+	{
+		super(connection);
+	}
+
+	private SnowflakeJdbcTransactionManager transactionManager;
+
+	@Override
+	public void beginTransaction()
+	{
+		try
+		{
+			this.transactionManager = new SnowflakeJdbcTransactionManager(connection);
+			this.transactionManager.beginTransaction();
+		}
+		catch (SQLException e)
+		{
+			throw new RuntimeException(e);
+		}
+	}
+
+	@Override
+	public void commitTransaction()
```

Comment:
Can we resuse these methods from parent JdbcHelper.
probably we can set  this to protected in JdbcHelper and that could work  :
 private JdbcTransactionManager transactionManager;